### ISSUE TITLE
Excessively long arrows/boxes when not specified

### ISSUE THREAD

**Issue by notching** [plantuml/plantuml#1181]
### Discussed in https://github.com/plantuml/plantuml/discussions/1179

<div type='discussions-op-text'>

<sup>Originally posted by **notching** November  3, 2022</sup>
The following uml seems to create excessively long arrow/boxes - any idea why?  Have upgraded sphinx / plantuml and am now encountering this issue.  Older versions work fine. 

```
.. uml::
    :caption: Looong Example
    :align: center

    @startuml

    rectangle "Box1" {
        rectangle "Box2" {
            rectangle "CCQ" as ccq
            rectangle "MRR" as mrr
            rectangle "CAS" as CA
            rectangle "AM" as am
            rectangle "KM" as km
            rectangle "DS" as ds
        }
        interface OUT
        ccq <-d-> OUT : To/from
        Supervisor <-d-> mrr
        mrr <-d-> CA
        mrr <-d-> ccq
        mrr <-d-> am
        mrr <-d-> km
        mrr <-d-> Internal
        mrr <-d-> ds
    }

    @enduml
```
Resulting image in pdf:
https://drive.google.com/file/d/1s9drP1zlXMA18SUzKX3iiIxjZsx7RZIx/view?usp=share_link </div>

**Comment by arnaudroques** [comment]
Which version of GraphViz are you using?
You may have to upgrade GraphViz.

**Comment by notching** [comment]
Trying to install graphviz gives:

`graphviz is already the newest version (2.40.1-2).`

Perhaps I need to build a later version from sources.

**Comment by arnaudroques** [comment]
So that's the issue: version 2.40 and 2.42 have issues (see https://forum.plantuml.net/4531/possible-bug-with-relation-in-rectangle-screenshot )
You should at least upgrade to 2.44

**Comment by notching** [comment]
Thanks, will give 2.44 a try. V6 resulted in QR codes everywhere.

**Comment by notching** [comment]
Thanks - fixed with v2.44 - which is a bit old now so I had to build from source. 
Then got other errors plantuml on colors:
`java.lang .IllegalStateException : Cannot find color # 0 0 0 0 0 7`

Which turned out to be not building with libexpat-dev installed and configured in the graphviz makefile.
